Documentation/ldap-connector.md

```diff
@@ -0,0 +1,99 @@
+# Authentication through LDAP
+
+## Overview
+
+The LDAP connector allows email/password based authentication, backed by a LDAP directory. All the configuration parameters required by the connector are held in [Config struct](../connector/ldap/ldap.go).
+
+The connector executes two primary queries:
+
+1. Finding the user based on the end user's credentials.
+2. Searching for groups using the user entry.
+
+## Example: User authentication
+
+User entries are expected to have an email attribute (configurable through `emailAttr`), and a display name attribute (configurable through `nameAttr`). The following is an example config file that can be used by the LDAP connector to authenticate a user.
+
+```yaml
```

Comment:
For inline comments, don't add a newline a the end of every sentence, add it add it at about 80 characters.

For example, don't do this:

```
# The DN and password for an application service account.
# The connector uses these credentials to search for users and groups.
# Optional. Not required if the LDAP server provides access for anonymous auth.
```

Do this:

```
# The DN and password for an application service account. The connector uses
# these credentials to search for users and groups. Not required if the LDAP
# server provides access for anonymous auth.
```